Cargo.toml

```
@@ -21,3 +21,5 @@ paste = "1"
 regex = { version = "1.4", default-features = false, features = ["std", "unicode-perl"] }
 strum = { version = "0.23", features = ["derive"] }
 unicode-normalization = "0.1"
+serde = { version = "1.0.133", features = ["derive"], optional = true }
```

Drop the patch version here 
```suggestion
serde = { version = "1.0", features = ["derive"], optional = true }
```